config/operator/operator_role.yaml

```diff
@@ -43,7 +43,13 @@ rules:
   - namespaces
   - persistentvolumeclaims
```

Comment:
Does hive operator deal with `persistentvolumeclaims`? Never heard of that. I might be wrong

Reply:
Opened HIVE-1920 to have a closer look at this and other things.